components/textdiff/textdiff.less

```diff
@@ -3,4 +3,12 @@
     padding: 10px 0;
     text-align: center;
   }
+
+  td.patched {
+    cursor: pointer;
+  }
+
+  .d2h-code-line-prefix > input[type='checkbox'] {
```

Comment:
Could this be moved to here so everything regarding the checkbox is in one place?
https://github.com/FredrikNoren/ungit/blob/3c80532c102be4b7227812faeb6bc3a330a2e8eb/components/staging/staging.less#L101-L105

Reply:
Sure, no problem (see 34f036a6dca58fc4f546f006a50751c13bad2ccc).